keras_rs/src/utils/pairwise_loss_utils.py

```diff
@@ -0,0 +1,100 @@
+from typing import Optional
+
+from keras import ops
+
+from keras_rs.src import types
+from keras_rs.src.utils.keras_utils import check_shapes_compatible
+
+
+def apply_pairwise_op(x: types.Tensor, op: ops) -> types.Tensor:
+    return op(
+        ops.expand_dims(x, axis=-1),
+        ops.expand_dims(x, axis=-2),
+    )
+
+
+def pairwise_comparison(
+    labels: types.Tensor,
+    logits: types.Tensor,
+    mask: types.Tensor,
+    logits_op: ops,
```

Comment:
The type annotation to `logits_op` seems wrong. Shouldn't it be a `Callable[[Tensor, Tensor], Tensor]`?

Reply:
Yep, need to change this